tests/fixtures/qasm2/shared15.qasm

```diff
@@ -1,7 +1,10 @@
 OPENQASM 2.0;
 include "qelib1.inc";
 gate iswap q0,q1 { s q0; s q1; h q0; cx q0,q1; cx q1,q0; h q1; }
+gate swap q0,q1 { cx q0,q1; cx q1,q0; cx q0,q1; }
+gate cp(theta) q0,q1 { cx q1,q0; u3(0, 0, theta) q0; cx q1,q0; }
```

Comment:
is this consistent with the shared15 circuits for the other program types?

Reply:
Again, added this when I was testing a change from pyqasm locally. Will revert.